GameServer.cpp

```diff
@@ -606,11 +600,6 @@ void GameServer::SendGameSessionRequest(sockaddr_in &to, unsigned int serverRand
 #ifdef WIN32
 	void GameServer::InitializeWinsock()
 	{
-		// Make sure Winsock was not already initialized
-		if (bWinsockInitialized != false) {
-			throw std::runtime_error("Winsock already enabled in GameServer");
-		}

```

Comment:
I like that this is no longer an error. There could be legitimate reasons to initialize and use Winsock outside of this class.